posthog/queries/breakdown_props.py

```diff
@@ -266,6 +267,8 @@ def _format_all_query(team: Team, filter: Filter, **kwargs) -> Tuple[str, Dict]:
     if entity and isinstance(entity, Entity):
         props_to_filter = props_to_filter.combine_property_group(PropertyOperatorType.AND, entity.property_groups)
 
+    prop_filter_out_keys(props_to_filter, ["$session_duration"])
```

Comment:
This here is particularly problematic - we're filtering out $session_duration because we're generally handling session filters incorrectly, right?

Currently there's only one session prop available on filtering, but what then happens when we start offering 15 more? Do we add them all here? Best would be to not have session properties get here in the first place I think

Reply:
Totally fair, I'll look into intervening further up 👍